Tests/Grid/Mapping/Metadata/ManagerTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping\Metadata;
+
+use APY\DataGridBundle\Grid\Mapping\Column;
+use APY\DataGridBundle\Grid\Mapping\Driver\Annotation;
+use APY\DataGridBundle\Grid\Mapping\Metadata\Manager;
+use APY\DataGridBundle\Grid\Mapping\Metadata\Metadata;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraints\DateTime;
+
+class ManagerTest extends TestCase
+{
+    public function testAddDriverWithStrings()
+    {
+        $manager = new Manager();
+        $manager->addDriver('foo', 'bar');
+        $driver = $manager->getDrivers();
+
+        $this->assertInstanceOf(
+            '\APY\DataGridBundle\Grid\Mapping\Metadata\DriverHeap',
+            $driver
+        );
+    }
+
+    public function testGetDriverReturnDifferentClone()
+    {
+        $manager = new Manager();
+        $manager->addDriver('foo', 'bar');
+
+        $driverFirstTime = $manager->getDrivers();
```

Comment:
Here you're testing `getter` and `adder`: same as above: do two different tests so we can be notified if failure is due to `add` or `get`